cmd/bundle/init.go

```diff
@@ -89,7 +89,7 @@ See https://docs.databricks.com/en/dev-tools/bundles/templates.html for more inf
 			if !cmdio.IsOutTTY(ctx) || !cmdio.IsInTTY(ctx) {
 				return errors.New("please specify a template")
 			}
-			templatePath, err = cmdio.Ask(ctx, "Template to use", "default-python")
+			templatePath, err = cmdio.AskSelect(ctx, "Template to use", []string{"default-python", "mlops-stacks"})
```

Comment:
Please use a unexported package variable for the list of default templates, then you can use it in the `Long` description of the command as well. This still says `default-python` is the only built-in template.

Reply:
Done. Added a couple of helper functions to generate the descriptions and dropdown options.